qualang_tools/results/results.py

```diff
@@ -136,4 +137,6 @@ def wait_until_job_is_paused(running_job: RunningQmJob, timeout: int = 30):
     while (not running_job.is_paused()) and (delay < timeout):
         time.sleep(0.1)
         delay = time.time() - start
+    if delay < timeout:

```

Comment:
shouldn't it be `delay > timeout`?
Also, we might want it to be an exception. Or at least, add a `strict_timeout` boolean argument, which if set to `true`, throws an exception.
WDYT?

Reply:
Oh wow very good point I was too tired I guess...
Yeah you're probably right, I wanted to minimize the arguments but throwing an exception by default is probably more robust and closer to what customers will want